collections/traitattr/occurattributes.php

```diff
@@ -49,360 +51,356 @@
 $start = $attrManager->getFilterAttribute('start');
 
 $statusStr = '';
-if($isEditor){
-	if($submitForm == 'Save and Next'){
+if ($isEditor) {
+	if ($submitForm == 'Save and Next') {
 		$attrManager->setOccid($_POST['targetoccid']);
-		if(!$attrManager->addAttributes($_POST,$SYMB_UID)){
+		if (!$attrManager->addAttributes($_POST, $SYMB_UID)) {
 			$statusStr = $attrManager->getErrorMessage();
 		}
-	}
-	elseif($submitForm == 'Set Status and Save'){
+	} elseif ($submitForm == 'Set Status and Save') {
 		$attrManager->setOccid($_POST['targetoccid']);
 		$attrManager->editAttributes($_POST);
 	}
 }
 $imgArr = array();
 $occid = 0;
 $catNum = '';
-if($traitID){
+if ($traitID) {
 	$imgRetArr = array();
-	if($mode == 1){
+	if ($mode == 1) {
 		$imgRetArr = $attrManager->getImageUrls();
 		$imgArr = current($imgRetArr);
-	}
-	elseif($mode == 2){
+	} elseif ($mode == 2) {
 		$imgRetArr = $attrManager->getReviewUrls($traitID);
-		if($imgRetArr) $imgArr = current($imgRetArr);
-
+		if ($imgRetArr) $imgArr = current($imgRetArr);
 	}
-	if($imgRetArr){
+	if ($imgRetArr) {
 		$catNum = $imgArr['catnum'];
 		unset($imgArr['catnum']);
 		$occid = key($imgRetArr);
-		if($occid) $attrManager->setOccid($occid);
+		if ($occid) $attrManager->setOccid($occid);
 	}
 }
 ?>
 <html>
-	<head>
-		<title>Occurrence Attribute batch Editor</title>
-		<link href="<?php echo htmlspecialchars($CSS_BASE_PATH, HTML_SPECIAL_CHARS_FLAGS); ?>/jquery-ui.css" type="text/css" rel="stylesheet">
+
+<head>
+	<title><?php echo $LANG['OCC_ATTRIBUTE_BATCH_EDIT'] ?> </title>
```

Comment:
Careful of spaces in HTML. I think you need to remove excess spaces between tags, like removing the space between > and < . Maybe check with Nikita on this. Check out the commit that I did to your branch to see where I removed spaces.

Reply:
Thank you for the information. I'll do that right away.